nesta/core/scripts/docker_recipes/py36_sklearn_amzn1_image/Dockerfile

```diff
@@ -1,25 +1,55 @@
 FROM amazonlinux:1
 
+RUN du -cxh --threshold=5M --max-depth=3 /
+
+RUN mkdir /root/.conda
+ENV PYTHONDONTWRITEBYTECODE=true
+ENV PYTHONIOENCODING=utf8
+
 # Install common dependencies
 RUN rm -rf /var/cache/yum/x86_64/latest
 RUN yum update -y
-RUN yum install sudo -y
+#RUN yum install sudo -y
 RUN yum install aws-cli -y
-RUN yum install zip -y
+#RUN yum install zip -y
 RUN yum install unzip -y
-RUN yum -y install findutils
+RUN yum install which -y
+RUN yum install findutils -y
+RUN yum install wget -y
+RUN yum install gcc -y
```

Comment:
Is `gcc` needed? I think conda generally installs it's own version
(applies to line 36 too).